src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -351,6 +360,9 @@ public abstract partial class DbDataReader : System.Collections.IEnumerable, Sys
         public abstract bool Read();
         public System.Threading.Tasks.Task<bool> ReadAsync() { return default(System.Threading.Tasks.Task<bool>); }
         public virtual System.Threading.Tasks.Task<bool> ReadAsync(System.Threading.CancellationToken cancellationToken) { return default(System.Threading.Tasks.Task<bool>); }
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int ordinal) { return default(System.Data.IDataReader); }
+        public void Close() { }
+        DataTable System.Data.IDataReader.GetSchemaTable() { return default(DataTable); }
```

Comment:
Nit: Sorry I didn't mention this before, but should `DataTable` be fully qualified with the namespace? It seems like all the other types in this file (and other ref files) use fully qualified names.

```c#
System.Data.DataTable System.Data.IDataReader.GetSchemaTable() { return default(System.Data.DataTable); }
```

Applies throughout this PR.